src/modules/bulmaModal.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable space-before-function-paren */
+/* eslint-disable func-names */
+/* eslint-disable camelcase */
+/* eslint-disable prefer-arrow-callback */
+/* eslint-disable import/prefer-default-export */
+// modal
+export class BulmaModal {
+  constructor(selector) {
+    this.elem = document.querySelector(selector);
+    this.close_data();
```

Comment:
The convention for JS variables would be `closeData`. You can rename your functions to follow that convention.